src/DiskIO/DiskDaemon/DiskdAction.cc

```diff
@@ -65,13 +65,13 @@ DiskdAction::Create(const Mgr::CommandPointer &aCmd)
 DiskdAction::DiskdAction(const Mgr::CommandPointer &aCmd):
     Action(aCmd), data()
 {
-    debugs(79, 5, HERE);
+    debugs(79, 5, "");
```

Comment:
I think you missed my point in the summary. These are things where the HERE removal collides with other cleanup changes necessary and make the 100% scripted not possible *for those lines*. The script needs to be able to be integrated with something that does those other changes. So at worst skip the files where it cannot handle the necessary changes - those are handled manually via the compile errors when HERE is gone, or later by how this script is integrated with source-maintenance to do both changes automatically at the same time.

HERE removal is part of the general code polishing / cleanup in Squid. The intended workflow is that branch authors can merge/rebase the master branch changes into theirs - when a conflict occurs they can take their code chunk over any cleanup chunk. Then run the source-maintenance over the resulting code and review the automated changes *before* building it.

All the cleanup polishing has the same problems impacting each other so calling one out of scope for the others is not going to avoid the problem, just shift it around until the collide again later with both being applied by source-maintenance - with the same net effect on branches.

I noted each collision point here to make it easier to see which files are imposing collisions on this script. We can omit them from the script and go ahead with the automated parts first and manual fixes later. Or we can focus on doing the manual changes first as much smaller cleanup patch(es) and let branches cope with those smaller collisions before following up with the bigger scripted part.